packages/bpk-component-ticket/src/BpkTicket.module.scss

```diff
@@ -41,20 +41,23 @@ $bpk-spacing-v2: true;
 }
 
 .bpk-ticket {
-  $notch-size: bpk-spacing-md();
   $ticket-padding: bpk-spacing-base();
 
   position: relative;
   display: block;
   display: flex;
   flex-direction: row;
   align-items: stretch;
+  order: 1;
+  flex: none;
+  flex-grow: 0;
   color: $bpk-card-color;
   text-decoration: none;
+  box-shadow: 0 5px 12px rgba($bpk-button-outline-background-color, 0.08),
```

Comment:
Should use backpack variable instead

Reply:
there is no relative value in backpack

Reply:
`$bpk-one-pixel-rem` would be better than raw `px`

Reply:
I think `px` in `box-shadow` has been considered available, we could also see the related rule in this [file](https://github.com/Skyscanner/backpack/blob/6677e26bef892f921390c52a9448a4ca6b60ef70/.stylelintrc.json#L10)